advanced-cache.php

```diff
@@ -487,16 +487,6 @@ function add_debug_html_to_output( $debug_html ) {
 
 wp_cache_init(); // Note: wp-settings.php calls wp_cache_init() which clobbers the object made here.
 
-global $wp_object_cache;
-if ( ! is_object( $wp_object_cache ) ) {
```

Comment:
you'll need to also change the method_exists below :)

Reply:
Done, thanks for pointing that out.